lists/eg.csv

```diff
@@ -155,3 +155,4 @@ https://www.facebook.com/ikhwanweb.official,FEXP,Free expression and media freed
 https://www.alaraby.co.uk,FEXP,Free expression and media freedom,2016-10-23,OONI,
 https://whispersystems.org,COMT,Communication Tools,2017-01-19,citizenlab,
 http://textsecure-service-ca.whispersystems.org,COMT,Communication Tools,2017-01-19,citizenlab,Used by Signal desktop,

```

Comment:
I would also remove this line since we would be testing something that is not reachable.